app/lib/actions/katello/system/bulk_auto_attach_subscriptions.rb

```diff
@@ -0,0 +1,34 @@
+module Actions
+  module Katello
+    module System
+      class BulkAutoAttachSubscriptions < Actions::EntryAction
```

Comment:
This class has no knowledge of how its being called :) This is a singular action item, that you can, call in a bulk way through the method you describe above, so I think it is better for the action to reflect what it does, not what something else could do with it.